p5-travel-app/src/client/index.js

```diff
@@ -6,9 +6,12 @@
 import { getGeoCity } from "./js/geoNamesApi";
 import { getWeatherForecast } from "./js/weatherApi";
 import { getCity, getTripStart, getTripEnd } from "./js/form";
+import { showModal, recentTrip } from "./js/model";
+import { getCountryInfo } from "./js/restCountriesApi";
 
 //import style
 import "./styles/style.scss";
+// import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
remove

Reply:
done, clean it in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)